src/services/legal-services.ts

```diff
@@ -182,21 +184,88 @@ export default class LegalServices {
    * @param businessId the business identifier
    * @returns a promise to return the parties from the response, else exception
    */
-  static fetchParties (businessId: string): Promise<any> {
+  static async fetchParties (businessId: string): Promise<any> {
```

Comment:
Another leetle bug fix. The impact would have been that code after fetchParties() would have run before THIS code completed.